packages/salesforcedx-vscode-core/src/commands/baseDeployRetrieve.ts

```diff
@@ -96,6 +88,27 @@ export abstract class DeployRetrieveExecutor<
     }
   }
 
+  async setApiVersionOn(components: ComponentSet) {
+    // Check the SFDX configuration to see if there is an overridden api version.
+    // Project level local sfdx-config takes precedence over global sfdx-config at system level.
+    const userConfiguredApiVersion = await this.getUserConfiguredApiVersion();
```

Comment:
Also the 'user' is maybe not the one who set this... maybe `projectConfiguredApiVersion`?

Reply:
Oh, will this still get the global setting? Maybe not adding "project" in then...

Reply:
Yeah, it will be local if there is one, and global if there isn't a local one set.  Either way I think it is always a user that sets the value that is found in config.  Could go with just configApiVersion I suppose, but wanted it to be clearly distinct from the orgApiVersion.